azure-steps-windows.yml

```
@@ -64,6 +65,7 @@ steps:
   displayName: 'Check OpenBLAS version'
 
 - powershell: |
+    $env:RTOOLS40_HOME = "c:rtools40"
```

```suggestion
    $env:RTOOLS40_HOME = "C:\rtools40\"
```
typo with missing slash